dcrjson/dcrwalletextcmds.go

```diff
@@ -532,6 +534,30 @@ func NewWalletInfoCmd() *WalletInfoCmd {
 	return &WalletInfoCmd{}
 }
 
+type OutputDestination struct {
+	Address       string
+	Script        []byte
+	ScriptVersion uint32
```

Comment:
we probably don't need the script and script version, and can just work with an address string instead of this type.  P2SH is a better idea anyways for using custom scripts